ignite/pkg/availableport/availableport.go

```diff
@@ -7,17 +7,74 @@ import (
 	"time"
 )
 
+type OptionalParameters struct {
```

Comment:
can you please create the options struct like the default pattern:

https://github.com/ignite/cli/blob/e9e6ff4c835687a0b756de9c19d032a9f749a8a5/ignite/services/scaffolder/module.go#L115-L173

Reply:
I adapted the code for this structure, but now I'm having a problem trying to make the tests work the way they should with this new formatting. I'll have to make an update on it once I see how to solve it.